pywemo/subscribe.py

```diff
@@ -35,32 +73,124 @@ def get_ip_address(host='1.2.3.4'):
 
 
 class RequestHandler(BaseHTTPRequestHandler):
-    """Handles subscription responses received from devices."""
-
-    def do_NOTIFY(self):
+    """Handles subscription responses and long press actions from devices.
+
+    Subscription responses:
+      Pywemo can subscribe to Wemo devices. When subscribed, the Wemo device
+      will send notifications when the state of the device changes. The
+      do_NOTIFY method below is called when a Wemo device changes state.
+
+    Long press actions:
+      Wemo devices can control the state of other Wemo devices based on the
+      rules configured for the device. A long press rule is activated whenever
+      the button on the Wemo device is pressed for 2 seconds. The long press
+      rule is meant to be used to control the state of another device (turn
+      on/off/toggle). However for pywemo's use, a long press rule can be used to
+      trigger an event notification. This is implemented by configuring the
+      Wemo device to "control the state" of a virtual Wemo device. The virtual
+      device is implemented by this class.
+
+      The do_GET/do_POST/do_SUBSCRIBE methods below implement a virtual Wemo
+      device. The virtual device receives requests to change its state from
+      other Wemo devices on the network. When a Wemo device is configured to
+      change the state of the virutal device via a long press rule the following
```

Comment:
typo on "virtual" here.  Also, a bit of a nitpick here but the configuration of black/isort have the line length set to 79 (the 80th character being the newline), but it looks like you have your right "rule" in your editor set at 80 characters as some lines in the comments go 1 extra character, including the "g" on "following" in this line.

Reply:
Fixed both the typo and the line length issue.